CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 2.8.0
+
+### Major changes
+
+- New [i18n](https://github.com/bem/bem-core/tree/v2/common.blocks/i18n) block was introduced, providing support for internationalization ([#1074](https://github.com/bem/bem-core/issues/1074)).
```

Comment:
As far as I understand, it's impossible to start using `i18n` on existing projects, with simple update of bem-core. Am I right that one has to change something in the service building process to use it? What kind of changes should be done; what new enb-modules should be added? 

I'd prefer to have at least a minimal docs before release, if there are no users who can't wait.